ch.imatt.ClipQR.yml

```diff
@@ -0,0 +1,42 @@
+app-id: ch.imatt.ClipQR
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+
+command: clipqr
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=all
+
+build-options:
+  build-args: [ "--share=network" ]
```

Comment:
this will fail the build. there is no network.

Reply:
Ok many thanks for your help, I will try without this :)